app/Http/Controllers/API/Posts/PostsController.php

```diff
@@ -85,6 +85,7 @@ public function store(Request $request)
     public function index(Request $request)
     {
         $filters = $this->demoCheck($this->getFilters($request));
+        $filters['limitUnprivileged'] = 1;
```

Comment:
I think we would actually always want to have limits. It's really not a good practice not to have them. In that case, this activation here is probably unnecessary.

Reply:
Ok I can do that. The way the issue was written it sounded like only non-privileged needed a limit.

Reply:
> Ok I can do that. The way the issue was written it sounded like only non-privileged needed a limit.

Yes, that's right, that's pretty much how it was written.  I think it's just that we have learned a thing or two as we have seen more traffic spikes 😅

Reply:
@tuxpiper I totally get having real world results changing your reqs. HOwever, there was also this comment in the issue:
"This limit should not affect other endpoints that deal with post data, such as /api/v3/geojson"
So it seems like we still need some kind of activation from this point only?

Reply:
Ah yea, it's definitely more of an endpoint dependent restriction, not something that should be driven from the model or usecase layer. For the GeoJSON endpoint there should also be limits (it's a very expensive query! once you get to 500+ posts) , but those could be different as the concerns are different.

Reply:
I'll rename the config vars to allow for adding separate values for other endpoints in the future.